servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -112,24 +106,36 @@
     Locale.setDefault(Locale.ENGLISH);
   }
 
-  protected void init(NessieClient client, HttpClient httpClient) {
-    this.client = client;
+  protected void init(URI uri) {
+    NessieApiV1 api =
+        HttpClientBuilder.builder().withUri(uri).build(NessieApiVersion.V_1, NessieApiV1.class);
```

Comment:
this part should be configurable so that we can just pass NessieApiV1 from subclasses to `init` as grpc stuff will use a different builder

Reply:
That's why there are 2 `init` methods:
`protected void init(URI uri)` which calls `protected void init(NessieApiV1 api, HttpClient httpClient)`

Reply:
ah sorry, totally missed that somehow